README.md

```diff
@@ -97,7 +97,7 @@ You can run the tests with the commands below:
 ```
 cd manticore
 # all tests
-nosetests
+nosetests --exe
```

Comment:
I'm on nosetests 1.3.7 and when i run it in the top level, it actually does run the tests for me! Also, my test files do not have the execute bit on them. i'm not sure what the issue is; could we omit this change for the time being?